cluster-scope/overlays/prod/common/alertmanager-main-secret.yaml

```
@@ -67,6 +67,9 @@ stringData:
       - name: Github
         webhook_configs:
           - url: 'http://github-receiver-service.opf-alertreceiver.svc.cluster.local:9393/v1/receiver'
+      - name: Thoth-Github
+        webhook_configs:
+          - url: 'http://github-receiver-service.thoth-infra-prod.svc.cluster.local:9393/v1/receiver'
```

```suggestion
          - url: 'http://github-receiver.thoth-infra-prod.svc.cluster.local:9393/v1/receiver'
```

I think URL won't have the `-service` 
https://github.com/thoth-station/thoth-application/blob/dd21dab477e85735b4d1b84eec0547d41f9b3430/monitoring/base/github-receiver.yaml#L5